autoprotocol/builders.py

```diff
@@ -27,13 +27,13 @@
 
 from collections import defaultdict
 from collections.abc import Iterable  # pylint: disable=no-name-in-module
-from dataclasses import dataclass
 from functools import reduce
 from numbers import Number
 from typing import Any, Dict, List, Optional, Union
 
 from .constants import SBS_FORMAT_SHAPES
 from .container import Container, Well, WellGroup
+from .types.protocol import *  # pylint: disable=unused-wildcard-import
```

Comment:
Super nit (optional):  I understood that it was for shortened import statement to use wildcard, but I wonder if listing out all the actual items being imported would be more readable? 
My personal perspective on using wildcard was:
1) May have impact on the intialization processor of the module during runtime if the types/protocol.py become large in the near future. 
2) Less readable for troubleshooting/debugging the classes being used in this module - not immediately telling us where those files come from ...
3) May be risky of colliding namespace of same classes could exist on two or more different libraries

But this is just my thought :-)